README.md

```diff
@@ -242,6 +242,9 @@ https_cert_file = ""
 // The client key file to use for talking to HTTPS checks.
 https_key_file = ""
 
+// Client address. Example: "127.0.0.1:8080"
```

Comment:
I'd like to add a little more information here so that users will know when they should configure this. Maybe something like: "Client address to expose API endpoints. Required in order to expose /metrics endpoint for Prometheus." Feel free to use any other language you prefer!

Reply:
I agree, I followed your language suggestion.